mscore/editstaff.ui

```diff
@@ -422,16 +512,71 @@
           </property>
          </widget>
         </item>
-        <item row="1" column="1">
-         <widget class="QTextEdit" name="shortName">
-          <property name="minimumSize">
-           <size>
-            <width>0</width>
-            <height>40</height>
-           </size>
+        <item row="3" column="0">
+         <widget class="QLabel" name="labelOffsetX">
+          <property name="toolTip">
+           <string>Negative values move left, positive values move right (-200sp, 200sp).</string>
           </property>
-          <property name="tabChangesFocus">
-           <bool>true</bool>
+          <property name="text">
+           <string>Name Offset X:</string>
+          </property>
+         </widget>
+        </item>
+        <item row="2" column="0">
+         <widget class="QLabel" name="rotationLabel">
+          <property name="toolTip">
+           <string>Negative values rotate counterclockwise, positive values clockwise (-180°...180°).</string>
+          </property>
+          <property name="text">
+           <string>Name rotation:</string>
+          </property>
+         </widget>
+        </item>
+        <item row="2" column="1">
+         <widget class="QSpinBox" name="rotationInput">
+          <property name="toolTip">
+           <string>Negative values rotate counterclockwise, positive values clockwise (-180°...180°).</string>
+          </property>
+          <property name="toolTipDuration">
+           <number>-1</number>
+          </property>
+          <property name="minimum">
+           <number>-180</number>
+          </property>
+          <property name="maximum">
+           <number>180</number>
+          </property>
+         </widget>
+        </item>
+        <item row="3" column="1">
+         <widget class="QSpinBox" name="spinBoxOffsetX">
+          <property name="toolTip">
+           <string>Negative values move left, positive values move right (-200sp, 200sp).</string>
```

Comment:
It will be great to have offsets, thanks for creating this PR.
Will these limits be enough for cutaway scores? Sometimes a huge horizontal indent is needed:
![image](https://user-images.githubusercontent.com/2843953/90357135-4938e300-e028-11ea-8209-0c08e82f2b4d.png)